Makefile.validation

```diff
@@ -144,3 +144,30 @@ kind-create-cluster:
 
 kind-delete-cluster:
 	$(KIND) delete cluster
+
```

Comment:
This code already in the master

https://github.com/dell/csi-baremetal/blob/master/Makefile.validation#L57-L108

Reply:
It is a way to pull sidecars from open docker registry. Now we have no this opportunity